### PR TITLE
[1LP][RFR] fixed test_group_crud_with_tag for vsphere55

### PR DESCRIPTION
**Pull request by lina-nikiforova**
{{ pytest: -v cfme/tests/configure/test_access_control.py::test_group_crud_with_tag --use-provider complete }}

### CODE REVIEW

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -410,19 +410,25 @@ def test_group_crud_with_tag(provider, tag_value, group_collection):
         * Save group
     """
     tag_for_create, tag_for_update = tag_value
+
+    if provider.key == 'vsphere55':
+        path = 'VM_Template-Folder'
+    else:
+        path = 'Discovered virtual machine'
+
```
Comment (jawatts) [inline cfme/tests/configure/test_access_control.py]: **Optional** The more pythonic way to write this statement would be :
```path = 'VM_Template-Folder' if provider.key == 'vsphere55' else 'Discovered virtual machine'```

**Comment by jawatts** [comment]
@lina-nikiforova since PRT is failing on 510z login, could you post the results of your local execution please.

**Comment by lina-nikiforova** [comment]
Since PRT is failing on 510z, posting local results ran on CFME 5.10.0.16:

```
collected 2 items                                                                                                                                                                                                                                                      
Uncollection Stats:
 manual: 0
 uncollectif: 0
 2 tests left after all uncollections

cfme/tests/configure/test_access_control.py::test_group_crud_with_tag[tag-virtualcenter] [WARNING] [py.warnings] /home/anikifor/cfme/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
 (/home/anikifor/cfme/lib/python2.7/site-packages/psycopg2/__init__.py:144)

Trying to set up provider vsphere55

[WARNING] [py.warnings] /home/anikifor/cfme/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py:788: DeprecationWarning: use driver.switch_to.alert instead
  warnings.warn("use driver.switch_to.alert instead", DeprecationWarning)
 (/home/anikifor/cfme/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py:788)

cfme/tests/configure/test_access_control.py::test_group_crud_with_tag[tag-virtualcenter] PASSED
cfme/tests/configure/test_access_control.py::test_group_crud_with_tag[tag_expression-virtualcenter] 
Trying to set up provider vsphere55


cfme/tests/configure/test_access_control.py::test_group_crud_with_tag[tag_expression-virtualcenter] PASSED

```

File: cfme/tests/configure/test_access_control.py (hunk 1 above)
Comment (lina-nikiforova) [reply cfme/tests/configure/test_access_control.py]: Thanks!

**Pull request merged**